apps/gitness/src/pages/sandbox-pull-request-list-page.tsx

```diff
@@ -55,7 +55,8 @@ function PullRequestSandboxListPage() {
           title="No Pull Requests yet"
           description={['There are no pull requests in this repository yet.']}
           primaryButton={{
-            label: 'Create pull requests'
+            label: 'Create pull requests',
+            to: `/sandbox/spaces/${spaceId}/repos/${repoId}/pull-requests/compare`
```

Comment:
relative path here as well maybe?

Reply:
still needed since relative links don't work rn as some pages are on sandbox and some are not